### ISSUE TITLE
libopenh264.so.0 missing in openvidu-server-kms Docker image

### ISSUE THREAD

**Issue by 0xFelix** [OpenVidu/openvidu#44]
The file `libopenh264.so.0` is missing in the openvidu-server-kms Docker image. Without it kurento fails to support H264 and therefore Safari clients fail to connect. This again leads to the somewhat widespread error `OperationError (DOM Exception 34): Expects at least 4 field` which was originally fixed by installing `openh264-gst-plugins-bad-1.5`.

The reason for the lib missing is that the packages `wget` and `bzip2` are missing. Without these the dependency `openh264` of `openh264-gst-plugins-bad-1.5` fails to download the shared library.

Adding the following line between lines 9 and 10 in `openvidu/openvidu-server/docker/openvidu-server-kms/Dockerfile` should fix this problem.

`&& apt-get -y install wget bzip \`

**Comment by pabloFuente** [comment]
Thanks for opening this issue.

We will look into it.

**Comment by 0xFelix** [comment]
It should be `bzip2` instead of `bzip`, sorry

`&& apt-get -y install wget bzip \`

**Comment by djhr** [comment]
Facing the same issue.

Related to:
https://github.com/Kurento/bugtracker/issues/186#issuecomment-384701838

More info about it:
https://docs.google.com/document/d/1gbhJxZEIzSQR5yHwWXgQw_UxCjhJSc8_BEmYgChP8DI/edit

**Comment by djhr** [comment]
Confirmed that installing `wget` & `bzip2` will solve the issue.
Tested with the following image:
```
FROM openvidu/openvidu-server-kms:2.5.0

RUN apt-get update \
 && apt-get install -y wget bzip2 \
 && apt-get install -y --reinstall openh264
```

**Comment by j1elo** [comment]
This is a known source of installation problems: [OpenH264 not found](https://doc-kurento.readthedocs.io/en/latest/user/troubleshooting.html#openh264-not-found)

but the fact that `wget` and `bzip2` are not installed as dependencies is a different thing; it's a bug in the Deb package. So I added `wget` as a dependency in Kurento/openh264@0d74e83dcc5d919851b181d32a5b888f9befee94 and now thanks to your observations I've also added `bzip2` in Kurento/openh264@5faa3d1383b41c0dc0279d7a37b9ca2b56ebf6b7.

Thanks!

These fixes will be included in the next Kurento release 6.8.1, expected for the next week. This will fix the issue in Kurento-based installations including OpenVidu.
